CHANGES.md

```diff
@@ -58,6 +60,181 @@
 * [Version 1.0.1](#version-101)
 * [Version 1.0](#version-10)
 
+## Version 3.1.6 (March 9, 2023)
+
+Patch release that address various bug/build issues and optimizations:
+
+* NEON optimizations for ZIP reading
+* Enable fast Huffman & Huffman zig-zag transform for Arm Neon
+* Support relative and absolute libdir/incluedir in pkg-config generation
```

Comment:
There's a small typo in includedir

Reply:
Thanks, #1355.  Unfortunately, the typo is also in the PR title, but I'll leave that as is.